Features2/tests/features/deleteContact.feature

```diff
@@ -0,0 +1,27 @@
+Feature:Delete Contact
+    As a user
+    I want to delete contact
+    So I can  remove unwanted contacts from the contacts saved
+    Background:
+        Given the user has navigated to homepage
+
+     Scenario Outline:To delete the contacts from the saved contacts
+        Given  the user has added following contacts
+        | name | Email | Phone | Contact Type |
+        | <name> |<Email>| <Phone>| <Contact  Type>|
```

Comment:
This can be shifted to Background as there seems to be repetition in both scenarios.